tests/fr/_fixtures.yaml

```
@@ -1,15 +1,15 @@
 language: fr
 areas:
-  - name: Cuisine
+  - name: cuisine
     id: kitchen
-  - name: Salon
+  - name: salon
     id: living_room
-  - name: Chambre
+  - name: chambre
     id: bedroom
-  - name: Garage
+  - name: garage
     id: garage
-  - name: "Entrée"
-    id: "hall"
+  - name: entrée
+    id: hall
 entities:
   - name: lumière de la chambre
     id: light.bedroom_lamp
```

```suggestion
  - name: lampe de chevet
    id: light.bedside_lamp
```
I think we can change this entity because `lumière de la chambre` is not a good example because it include the domain and the area name.

---

I finally refactored a major part (see my last comment)